src/main/java/frc/robot/subsystems/vision/VisionConstants.java

```diff
@@ -7,7 +7,19 @@
 
 package frc.robot.subsystems.vision;
 
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Transform2d;
+import edu.wpi.first.math.geometry.Translation2d;
+
 public final class VisionConstants {
   public static final String noteDetectionLimelight = "limelight-note";
   public static final String centerLimelight = "limelight";
+
+  public static final double noteLimelightAngleDegrees = 0; // TODO: idk tune
+  public static final double noteLimelightHeightInches = 10; // TODO: idk tune
+  public static final double noteHeight = 2;
+
+  public static final Transform2d robotToCam =
```

Comment:
the camera is a non-insignificant distance away from the center of the bot, but this can be determined via CAD, ping aimee/sid for help with this

Reply:
Yea just waiting for the CAD to be finished